SAS/ttest.qmd

```diff
@@ -0,0 +1,78 @@
+---
+title: "Students t-test"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+### **Independent t-test in SAS**
+
+The null hypothesis of the Independent Samples t Test is, the means for the two populations are equal.
+
+In SAS the following code was used to test the mean comparison (mean of Weight Gain) of two independent treatment groups (Treatment and Placebo).
+
+For this example, we're testing the significant difference in mean of Weight gain (*WtGain*) between treatment and placebo (*trt_grp*) using PROC TTEST procedure in SAS.
+
+```{r}
+#| eval: false 
+  proc ttest data=d1; 
+     class trt_grp; 
+     var WtGain; 
+  run; 
+```
+
+Output:
+
+```         
+             Figure 1: Test results for independent t-test using PROC TTEST in SAS
+```
+
+```{r, echo=FALSE, fig.align='center', out.width="50%"}
+knitr::include_graphics("../images/ttest/test1.png")
+```
+
+Here the t value is --0.70, degrees of freedom is 30 and P value is 0.4912 which is greater than 0.05, so we accept the null hypothesis that there is a significant difference between the means of treatment groups. Also the mean difference is --7.9375 and the 95% CI for the mean difference is [--31.1984, 15.3234].
+
+Note: Before entering straight into the t-test we need to check whether the assumptions (like the equality of variance, the observations should be independent, observations should be normally distributed) are met or not. If normality is not satisfied, we need to go with the non-parametric test of independent t test.
+
+1.  Normality: We have checked the normality of the observations using the code below. Here for both the treatment groups we have P value greater than 0.05 (Shapiro-Wilk test is used), therefore the normality assumption is there for our data.
```

Comment:
Please change:  "1. Normality: We have checked"  to be:
"1. Normality: You can check for data to be normally distributed by plotting a histogram of the data by treatment. Alternatively, you can use the Shapiro-Wilk test or the Kolmogorov-Smirnov test.  If the test is <0.05 and your sample is quite small then this suggests you should not use the t-test. However, if your sample in each treatment group is large (say >30 in each group), then you do not need to rely so heavily on the assumption that the data have an underlying normal distribution in order to apply the two-sample t-test.  This is where plotting the data using histograms can help to support investigation into the normality assumption.  We have checked"......